package.json

```diff
@@ -46,10 +46,10 @@
     "classnames": "^2.2.6",
     "color": "^3.1.2",
     "compression": "^1.7.4",
-    "date-fns": "^2.11.1",
+    "date-fns": "^2.28.0",
```

Comment:
Have you done research on these changes? cml.dev is still at Gatsby 2 so we probably should be more cautous when we're upgrading packages. We really should get cml.dev to Gatsby 4... I'll start working on that after I finish my current P1.

Reply:
argh srry yes was uning `npm` instead of `yarn` and for once missed the double lockfile warning :facepalm: